modules/nf-core/fqtk/main.nf

```diff
@@ -0,0 +1,49 @@
+process FQTK {
+    tag "$meta.id"
+    label 'process_high'
+
+    conda "bioconda::fqtk=0.2.1"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/fqtk:0.2.1--h9f5acd7_0' :
+        'quay.io/biocontainers/fqtk:0.2.1--h9f5acd7_0' }"
+
+    input:
+    tuple val(meta), path(sample_sheet), val(fastq_readstructure_pairs)
+    // fastq_readstructure_pairs example:
+    // [[<fastq name: string>, <read structure: string>, <path to fastqs: path>], [example_R1.fastq.gz, 150T, ./work/98/30bc..78y/fastqs/]]
+
+    output:
+    // Demultiplexed file name changes depending on the arg '--output-types'
```

Comment:
This could probably be handled like https://github.com/nf-core/modules/blob/afb023c4e24892a76a05f6f448dbd3b630177e08/modules/nf-core/bedtools/closest/main.nf#L25-L27 in a future version.

Reply:
@Emiller88 thank you! I am happy to open an issue/PR after merge or fix now, which would you prefer?

Reply:
Up to you!

I don't see this as blocking adding it to demultiplex, and I'd rather see it in demultiplex first personally.

Reply:
Awesome, I will wait then.